src/platform/silabs/SiWx917/BUILD.gn

```diff
@@ -77,7 +75,17 @@ static_library("SiWx917") {
     "SiWxPlatformInterface.h",
   ]
 
-  if (chip_enable_ota_requestor) {
+  if (chip_enable_multi_ota_requestor) {
+    sources += [
+      "${silabs_platform_dir}/multi-ota/OTAHooks.cpp",
+      "${silabs_platform_dir}/multi-ota/OTAMultiImageProcessorImpl.cpp",
+      "${silabs_platform_dir}/multi-ota/OTAMultiImageProcessorImpl.h",
+      "${silabs_platform_dir}/multi-ota/OTATlvProcessor.cpp",
+      "${silabs_platform_dir}/multi-ota/OTATlvProcessor.h",
+      "${silabs_platform_dir}/multi-ota/SiWx917/OTAWiFiFirmwareProcessor.cpp",
+      "${silabs_platform_dir}/multi-ota/SiWx917/OTAWiFiFirmwareProcessor.h",
+    ]
```

Comment:
We should move these GN changes to a `BUILD.gn`  within the multi-ota directory and have a depency in the efr32 and SiwX917.